src/Plugin/Hal.php

```diff
@@ -330,26 +334,6 @@ public function setPaginationInjector(PaginationInjectorInterface $injector)
     }
 
     /**
-     * @param ServerUrl $helper
-     * @return self
-     */
-    public function setServerUrlHelper(ServerUrl $helper)
-    {
-        $this->serverUrlHelper = $helper;
-        return $this;
-    }
-
-    /**
-     * @param Url $helper
-     * @return self
-     */
-    public function setUrlHelper(Url $helper)
-    {
-        $this->urlHelper = $helper;
-        return $this;
-    }
```

Comment:
This is a BC break, unfortunately. I think it's likely okay, as it's undocumented functionality, and mainly to serve internal processes, but if anybody was supplying their own helpers previously, this now won't work.

When I merge, I'll re-instate these, but have them throw an exception, indicating what changes they will need to make.